test/test_jit_expression.py

```diff
@@ -141,3 +142,59 @@ def test_rank1(compile_args):
     u_correct = np.array([f[1], f[0]]) + gradf0
 
     assert np.allclose(u_ffcx, u_correct.T)
+
+
+def test_elimiate_zero_tables_tensor():
```

Comment:
Because @garth-wells  was wondering if they were necessary for the test to run, which they are not.
If you have a good reason for having them here, I will add them back in.